src/java.base/share/classes/java/lang/Long.java

```diff
@@ -1178,6 +1223,14 @@ private LongCache() {}
                 archivedCache = c;
             }
             cache = archivedCache;
+
+            HEX256 = new char[256];
+            for (int i = 0; i < 256; i++) {
+                int hi = (i >> 4) & 15;
+                int lo = i & 15;
+                HEX256[i] = (char) (((hi < 10 ? '0' + hi : 'a' + hi - 10) << 8)
+                        + (lo < 10 ? '0' + lo : 'a' + lo - 10));
+            }
```

Comment:
Did you mean to put this in LongCache and is the intention it be archived or are you putting this into its own holder class? Right now it's confusing as HEX256 is not read from the archive.

Reply:
i have already used a separate holder class "DigitCache", and modifiers do not use private because I will submit a new optimization using HEX256 for Integer # toHexString and Long # toHexString